spread-tests/hello-world-is-confined/task.yaml

```diff
@@ -0,0 +1,13 @@
+summary: Check that hello-world.evil is confined
```

Comment:
Reading the logic, this might be "Check that confinement works", and the test name might be "confined-blocked" or similar.